src/Page/Events/Event_.elm

```diff
@@ -113,7 +113,11 @@ viewEventInfo event =
         , case event.maybeGeo of
             Just geo ->
                 div [ css [ mapContainerStyle ] ]
-                    [ p [] [ Theme.Global.mapImage { latitude = geo.latitude, longitude = geo.longitude } ]
+                    [ p []
+                        [ Theme.Global.mapImage
+                            ("A map showing the location of " ++ event.name)
```

Comment:
I'd like all strings to go through the `Copy.Text.t` function. 

This includes alt text. I won't flag all of those up in this PR individually.

This 1) makes it easier to localise & 2) makes it easy to edit UI copy in one place (good overview / visibility on public facing content)

Reply:
I did wonder about this. Happy to do this, sure.

Reply:
This should be done now.